leaderboard/enriching/enricher.go

```diff
@@ -30,6 +31,15 @@ type (
 
 		// WebhookTimeout is the timeout for the webhook call.
 		WebhookTimeout time.Duration `mapstructure:"webhook_timeout,default=2s"`
```

Comment:
I do not think this `default=2s` works, does it?

Reply:
Indeed it doesn't. Removed.